controllers/entriesController.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+const User = require('../models/user');
+const Entry = require('../models/entry');
+
+// Get all Entries
+module.exports.listEntries = async (ctx, next) => {
+  if ('GET' != ctx.method) return await next();
+  const user = await User.findOne({'_id': ctx.user._id});
```

Comment:
You can take advantage of the authorization process (that's already loading the user) not to repeat the fetch to the db again. Store it in the context entirely.